manifests/g/GLab/GLab/1.19.0/GLab.GLab.yaml

```diff
@@ -0,0 +1,31 @@
+PackageIdentifier: GLab.GLab
+PackageName: GLab
+Moniker: glab
+PackageVersion: 1.19.0
+Publisher: GLab
+Author: Clement Sam
+License: MIT
+LicenseUrl: https://github.com/profclems/glab/blob/HEAD/LICENSE
+MinimumOSVersion: 10.0.0.0
+PackageUrl: https://github.com/profclems/glab
+ShortDescription: An Open source GitLab Command Line Tool.
+Tags:
+- GitLab
+- cli
+- glab
+- command-line
+- command line
+- commandline
+- utility
+- tool
+InstallerType: inno
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/profclems/glab/releases/download/v1.19.0/glab_1.19.0_Windows_x86_64_installer.exe
+  InstallerSha256: 9523631AF4D3B2FF756E4B085C829CAFBE2FE9F6F4B4AE5995C46A622FAC4D28
+  InstallerSwitches:
+    Silent: /VERYSILENT /ALLUSERS
+    SilentWithProgress: /SILENT /ALLUSERS
```

Comment:
```suggestion
```
I believe these aren't needed as winget-cli automatically runs /SILENT on inno installers?

Reply:
As far as i could see the installer didn't require /ALLUSERS either